cityofaustin/vision-zero
# Create locations pages and map

## Associated issues

* https://github.com/cityofaustin/atd-data-tech/issues/19960
* https://github.com/cityofaustin/atd-data-tech/issues/19975
* https://github.com/cityofaustin/atd-data-tech/issues/19977
* https://github.com/cityofaustin/atd-data-tech/issues/19976

## Testing

**URL to test:** Local

1. You will need to delete the `crashesListViewQueryConfig` entry from your localstorage
2. Use the sidebar to navigate to the locations page
3. Search by location ID
4. search by the location name
5. toggle the advanced filter switches on/off
6. sort and paginate through results
7. Refresh the page to see that your search is persisted
8. Navigate to a location details page
9. Verify that the location map loads correctly and is zoomed on the location polygon
10. See that the card to the right of the map displays the location ID

---

#### Ship list
